Puzzlepart/prosjektportalen365
# Risk matrix custom cells

### Your checklist for this pull request


### Description

This change makes it possible to add custom cells to the risk matrix. Colors, headers, text and number of cells can be modified to the organizations needs. 
A json (txt) file is fetched from the PP hubsite, stored in siteassets (https://{tenant}.sharepoint.com/sites{hubsite}/SiteAssets). The config file have to be named custom-cells.txt and have a certain format. A sample file is provided in the Risk-matrix code. 

### How to test

1. Upload the custom-cells.txt into the PP-instance siteassets.
2. Change the cell-color or header names. (or add some cells)
3. Notice the changes
4. Remove the config file, and the matrix should be rendered as usual. 


### Update of documentation
- [X] Check if user manual requires update
    - Update user manual


### Relevant issues (if applicable)

💔Thank you!
